updating/updating-cluster-within-minor.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update, or upgrade, an {product-title} cluster by using the web console. The following steps are updating a cluster within a minor a version. You can use the same instructions for updating a cluster between minor versions.
+You can update, or upgrade, an {product-title} cluster by using the web console. The following steps are updating a cluster within a minor version. You can use the same instructions for updating a cluster between minor versions.
```

Comment:
Suggestion: Change "...steps are updating a cluster..." to "...steps update a cluster..."

Reply:
Addressed